prac_05/wimbledon.py

```diff
@@ -0,0 +1,43 @@
+def read_wimbledon_data(filename):
+    champions = {}  # Dictionary to store champions and their win counts
+    countries = set()  # Set to store countries of champions
+
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        # Skip the header line
+        next(in_file)
```

Comment:
I don't recall using next as of yet, alas readline I imagine achieves the same thing.

Reply:
That is a good pickup, I completely forgot about that and have adjusted the code to 
in_file.readline()
 Thank you

Reply:
The comment above `next` is helpful :)